src/box/box.cc

```diff
@@ -5926,6 +5930,7 @@ box_storage_shutdown()
 	if (!is_storage_initialized)
 		return;
 	iproto_shutdown();
+	fiber_shutdown();
```

Comment:
Shouldn't we call it even if `is_storage_initialized` is unset? That is, shouldn't this be moved to `box_shutdown`?

Reply:
Added a comment as discussed f2f.

```diff
--- a/src/box/box.cc
+++ b/src/box/box.cc
@@ -5930,6 +5930,10 @@ box_storage_shutdown()
        if (!is_storage_initialized)
                return;
        iproto_shutdown();
+       /*
+        * Finish client fibers after iproto_shutdown otherwise new fibers
+        * can be started through new iproto requests.
+        */
        fiber_shutdown();
        replication_shutdown();
 }
```